content/en/governance/working-groups.md

```diff
@@ -6,11 +6,13 @@ weight: 100
 
 ## Working Groups
 
-Open Modeling Foundation Working Groups coordinate key activities, enabling a diverse spectrum of modeling scientists to participate directly in OMF activities and self-governance and ensuring that it can fulfill its vision, mission, and goals. Three Working Groups are initially established and described below; additional Working Groups may be established with approval of the Members Council.
+Open Modeling Foundation Working Groups coordinate key activities, enabling a diverse spectrum of modeling scientists to participate directly in OMF activities and self-governance and ensuring that it can fulfill its vision, mission, and goals. Five Working Groups are established and described below; additional Working Groups may be established with approval of the Members Council.
 
-Any modeling scientist can join a Working Group. Working Groups self organize, with support from the Executive Committee, and nominate a Working Group Chair for approval by the Members Council. Working Group Chairs serve for a three-year, renewable term.
+Any modeling scientist can join a Working Group. Working Groups self organize, with support from the Executive Committee, and nominate a Working Group Chair for approval by the Members Council. Working Group Chairs serve for a three-year, renewable term. Each Working Group may also choose to nominate any number of Co-Chairs as appropriate - each with the same term length and responsibilities as the Chair.
 
-Working Groups meet as needed at least once per year either virtually or in person.  
+Working Groups self-organize and meet as needed at least once per year either virtually or in person.  
+
+Working Groups are led by their chair(s), nominated by the Executive Committee, and approved by the Members Council. Working Groups can be led by multiple co-chairs. However, each Working Group only has a single vote within the Executive Committee.
```

Comment:
this verbiage still doesn't address #465 but perhaps it is best left to each WG to determine how to allocate their single vote?